subsys/testsuite/coverage/coverage.c

```diff
@@ -232,6 +232,48 @@ size_t gcov_to_gcda(uint8_t *buffer, struct gcov_info *info)
 	return buffer_write_position;
 }
 
+void gcov_reset_counts(struct gcov_info *info)
+{
+	struct gcov_fn_info *functions;
+	struct gcov_ctr_info *counters_per_func;
+	uint32_t iter_functions;
+	uint32_t iter_counts;
+	uint32_t iter_counter_values;
+
+	for (iter_functions = 0U;
+		iter_functions < info->n_functions;
+		iter_functions++) {
+
+		functions = info->functions[iter_functions];
+		counters_per_func = functions->ctrs;
+
+		for (iter_counts = 0U;
+			iter_counts < GCOV_COUNTERS;
+			iter_counts++) {
+			for (iter_counter_values = 0U;
+				iter_counter_values < counters_per_func->num;
+				iter_counter_values++) {
+				counters_per_func->values[iter_counter_values] = 0;
+			}
+		}
+	}
+}
+
+void gcov_reset_all_counts() {
```

Comment:
`checkpatch` can help to visualize this kind of issue from the vscode editor itself.

https://github.com/ycsin/zephyr/blob/ba9cc24441b50d9cde5254bde52b76748e119e2d/.devcontainer/devcontainer.json#L10-L43